tools/sync/sync.py

```diff
@@ -98,8 +98,8 @@ def worker(base):
                         help='config file in JSON format')
     parser.add_argument('-I', '--indexed', action='store_true',
                         help='Sync indexed projects only')
-    parser.add_argument('-m', '--messages',
-                        help='path to the Messages binary')
+    parser.add_argument('-u', '--url',
```

Comment:
add a sane default based on `localhost` ?

Reply:
What is a proper default? `localhost:8080`, `localhost:8080/source`, `localhost`, `localhost/source`?

Reply:
Also there is problem with the schema. If I use just `localhost` then `requests` don't work. `http://` or `https://` needs to be specified :/